pkg/exchange/coinbase/stream_handlers.go

```diff
@@ -262,8 +298,14 @@ func (s *Stream) handleTickerMessage(msg *TickerMessage) {
 	if !s.checkAndUpdateSequenceNumber(msg.Type, msg.ProductID, msg.Sequence) {
 		return
 	}
-	trade := msg.Trade()
-	s.EmitTradeUpdate(trade)
+	bookTicker := types.BookTicker{
+		Symbol:   toGlobalSymbol(msg.ProductID),
+		Buy:      msg.BestBid,
+		BuySize:  msg.BestBidSize,
+		Sell:     msg.BestAsk,
+		SellSize: msg.BestAskSize,
+	}
+	s.EmitBookTickerUpdate(bookTicker)
```

Comment:
just a remind: book ticker channel is for real-time ticker

Reply:
According to the coinbase doc, it is real-time ticker(price updates) every time a match happens:
https://docs.cdp.coinbase.com/exchange/docs/websocket-channels#ticker-channel